SpatialOS/scripts/download_dependencies.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
Some shell scripts use "-" in their names (like build-project.sh) while this one here uses "_". Can we make this consistent?

Reply:
done